sys/include/ztimer/xtimer_compat.h

```diff
@@ -74,6 +74,11 @@ static inline void xtimer_sleep(uint32_t seconds)
     ztimer_sleep(ZTIMER_USEC, seconds * 1000000LU);
 }
 
+static inline void xtimer_msleep(uint32_t milliseconds)
+{
+    ztimer_sleep(ZTIMER_MSEC, milliseconds);
```

Comment:
This requires the `ztimer_msec` module to work. We could either use some magic to fall back to `ZTIMER_USEC` if that module is unused, or extend `Makefile.dep`

Reply:
Ah I knew it wouldn't be that straightforward :laughing:

Reply:
msec should always work. I think @fjmolinas proposed to make that depend on ztimer_usec (which it will then transparently use), if ztimer_periph_rtt is not used.